### PR TITLE
Updating packed translations - v3.5

### PR DESCRIPTION
**Pull request by horus68**
Details
-------
- Global translation updates for BOLT v3.5 packed languages
- All languages files has passed by the lint tool from Bolt version 3.5 plain install

### CODE REVIEW

**Comment by horus68** [comment]
Note: Solves all questions and concerns placed in the now closed PR 7466

File: app/resources/translations/cs_CZ/messages.cs_CZ.yml
Comment (GwendolenLynch) [inline app/resources/translations/cs_CZ/messages.cs_CZ.yml]: Key names like this one `contenttypes.generic.actions-all` should not be quoted, as you'll see below the removed line is:

```diff
-contenttypes.generic.actions-all: "Akce pro %contenttypes%"
```

**Review by GwendolenLynch — CHANGES_REQUESTED** [review]

File: app/resources/translations/cs_CZ/messages.cs_CZ.yml
Comment (horus68) [reply app/resources/translations/cs_CZ/messages.cs_CZ.yml]: This is an action done by Bolt and it will do it for any edited/saved language file.
This was previous explained in this comment: https://github.com/bolt/bolt/pull/7466#discussion_r186095292

So this shouldn't be a block issue for this PR.
What will be the proper action here?

Comment (GwendolenLynch) [reply app/resources/translations/cs_CZ/messages.cs_CZ.yml]: > This is an action done by Bolt and it will do it for any edited/saved language file.
> This was previous explained in this comment: #7466 (comment)

Your comment is and was incorrect, it is the process you're using causing you issues.

> So this shouldn't be a block issue for this PR.

Yes, it does. I've spent the last couple of hours working on this, and as stated in teh beginning, having this done this way only creates a huge amount of work for us. This PR is **not** reviewable as-is, so I am trying to do things to help you … how about making that a two-way streeet, OK?

Comment (horus68) [reply app/resources/translations/cs_CZ/messages.cs_CZ.yml]: I'm here to help. I'm trying to understand this.
I will do a test with a plain install and will report back regarding those strings

**Comment by horus68** [comment]
I repeated the tests with the english file from 3.5 download and the results are the same i had before.
Original file:
```
#   0 Unused messages
#   2 Legacy untranslated messages
#   0 Untranslated messages
#  38 Legacy translation messages
# 558 Translation messages
```

Final file:
```
# 383 Unused messages
#   2 Legacy untranslated messages
#   0 Untranslated messages
#  23 Legacy translation messages
# 205 Translation messages
```

So please explain to me what am I doing wrong.
This was my workflow for a new language:

1- I'm using Wamp in a local install on Windows. Bolt is a 3.5 installed from a ZIP with "Flat structure"
Bolt is installed in F:\wamp\www\bolt35

1- Duplicated the en_GB folder and renamed folder and files to a new language so I can create a new language to test the issues. I used for the language code en_UD.
So they are the same files only with a different language code for name.

2- Placed the new folder in 

3- Opened Bolt admin, went to "Configuration/Translations: Messages" to edit the file.
Note: I will not perform any edit, just to see what bolt do with the file!

4- As soon the file is loaded you find Bolt has edited the file and the unused strings are now quoted.
See picture:
newlanguage_04

The file is properly saved into 
`F:\wamp\www\bolt35\vendor\bolt\bolt\app\resources\translations\en_UD`

As you can see the unused messages are now 383 and all that strings are quoted 
example:
`"contenttypes.generic.actions-all": "Actions for %contenttypes%"`
is now 
`contenttypes.generic.actions-all: "Actions for %contenttypes%"`

So this error is not done by me (as in a process made up by me) but by Bolt system,
I assume this will happen to anyone that try to edit its language file using Bolt interface (at least with a flat structure under windows.)

@GawainLynch can you replicate this behaviour? Is there a solution for this?

**Comment by GwendolenLynch** [comment]
OK, cleaned up. Which incidentally removed 2/3s of the PR and makes it easier to review.

**Comment by GwendolenLynch** [comment]
For what it is worth, this is what I get following those exact step here:
image

So yes, I would reassert there is something broken in your process.

**Comment by horus68** [comment]
Your image says it is working for you as it was supposed to be.
My image shows its not working for me.
Would love to have other users to replicate the tests as I don't have a solution.

**Comment by horus68** [comment]
I have more 14 new languages translated to Bolt but I think its better to not submit as a PR as I don't want to create more issues to you.
If you want I can send the files without been edited by Bolt so you can include them.

FYI:  The languages are: 
Arabic / Arabic Egypt / Azerbaijani / Cebuano / Filipino / Igbo / Nigerian Pidgin / Romanian  / Tagalog / Turkish / Ukrainian /  Urdu (India) / Urdu (Pakistan) /  Yoruba /

**Comment by GwendolenLynch** [comment]
I'm working on some tooling now to help ease the process. 

The problem is, as I have maintained for a long time, trying to do this without proper integration to the upstream platform just makes everyone's life harder. Albeit, we totally get the people doing the translations should have a pleasant and easy interface to do the job **they** need to do. 

Give me a couple of hours though, and I'll post an update here soon.

**Comment by horus68** [comment]
In the end the problems we have faced were created by my Bolt install not by an external tool!

But a tool to integrate translator works in the platform is the way. Hope you can find a way (take your time!)

**Comment by GwendolenLynch** [comment]
https://github.com/bolt/core-developer-tools

**Comment by GwendolenLynch** [comment]
> But a tool to integrate translator works in the platform is the way. Hope you can find a way (take your time!)

We can't with Crowdin … that is why we objected to it from the very start, but you went your own way. I have Tranifex working well with the v4 branch, and that is the way we'll go in future as the whole translations system is getting ripped out and replaced with Symfony's

**Comment by horus68** [comment]
To clear this thing up and to avoid ongoing discussions (its now some years long).
I don't have any problem to work in Transifex or any other tool. I'm also a long time user in Transifex with many projects as translator or as manager. See `https://www.transifex.com/user/profile/horus68/`
Translators always wanted a simple tool to work, and when things was not implemented we find our way. until a decision could be made (this was some years ago).
So we found another solution as a temporary solution. It worked for us, no problem for us as we can download our translations and use it in the system without any additional work: download/ unzip / done. The only problem here is on sending our work to the project. So I will wait for any solution to be implemented.

When you get Transifex working for translators I will use it, no problem. Any translator can then send their work to Transifex or any other tool the Bolt Project decide. I can do it for all the work done until that decision too! There will no problem for you!
So don't see me as a cause of problems. Find your way, we will follow.

**Comment by horus68** [comment]
As for Transifex: Is this to be used by translators? https://www.transifex.com/bolt/bolt-core/

**Comment by bobdenotter** [comment]
> As for Transifex: Is this to be used by translators? https://www.transifex.com/bolt/bolt-core/

It might be. If I recall correctly, Gawain and I set that up to _test_ it. But, as Gawain mentioned, translations will be pretty different in version 4, so we shouldn't spend too much time on that outdated test at transifex. When we get to the point that V 4 is solidified, we can make final decisions about what service to use.

**Comment by bobdenotter** [comment]
First thing I notice in the Dutch translation, is that it's confusing "U" (formal "you") and "Je" (informal "you"). It's a weird dutch thing, but we have to deal with it. Sometimes even within the same line: 

screen_shot_2018-05-09_at_14_36_19


(Note: this was already present.. Not introduced by you, @horus68 )

This, however is not a correct change: 

screen shot 2018-05-09 at 14 40 01

This is a revert of something I changed deliberately for 3.5:

screen shot 2018-05-09 at 14 40 58

This has been reverted in all files, it seems:  

screen shot 2018-05-09 at 14 42 29

Here's anothe case where a previously translated line gets reverted: 

screen shot 2018-05-09 at 14 44 19

screen shot 2018-05-09 at 14 44 32

I'm not skilled enough in other languages, but I see the same thing happening there. For example in Swedish: 

screen shot 2018-05-09 at 14 49 15
screen shot 2018-05-09 at 14 49 27

I can (and would not mind doing so) re-apply the changes I made for the `move-up` and `move-down`, as well as make the formal/informal "you" consistent (It should be informal, everywhere).

But, since I'm not confident there are no regressions in other translations that I can't overlook, I'd say that this PR needs work before we can merge it.

**Comment by GwendolenLynch** [comment]
9 hours work in two days for me to get this PR in shape isn't a cause of problems?

Reality check!

**Comment by horus68** [comment]
On dutch: There has been some translations changed online by some user.
On this PR: I will close it and lets forget about it for now.

**Pull request closed without merging**
